src/main/java/com/vmware/vipclient/i18n/base/cache/MessageCacheItem.java

```diff
@@ -73,18 +73,8 @@ public synchronized boolean isExpired() {
     	if (VIPCfg.getInstance().getCacheExpiredTime() != 0) {
     		return false;
     	}
-    	
-    	Long responseTimeStamp = this.getTimestamp();
-    	if (responseTimeStamp == null) {
-    		return true;
-    	}
-    	
-    	Long maxAgeResponse = this.getMaxAgeMillis();
```

Comment:
I don't agree. By setCacheItem, it can be null.

Reply:
You are correct! I already have listed it as todo: _Code clean up -  remove MessageCacheItem.setCacheItem (MessageCacheItem cacheItem)_

Reply:
Do you mean the timestamp property? Its type is long, not Long. The default value of long is 0. It cannot be null because it is not a Java object.

Reply:
I mean maxAgeMillis. It's Long type.

Reply:
How can it be null? There are 3 setCacheItem methods: lines 33, 38 and 46. All of them eventually execute lines 42-43:
`if (maxAgeMillis != null)
			this.maxAgeMillis = maxAgeMillis;`